docs/core/event_handler/bedrock_agents.md

```diff
@@ -0,0 +1,330 @@
+---
+title: Agents for Amazon Bedrock
+description: Core utility
+---
+
+Create [Agents for Amazon Bedrock](https://docs.aws.amazon.com/bedrock/latest/userguide/agents.html#agents-how){target="_blank"} using event handlers and auto generation of OpenAPI schemas.
+
+<center>
+```mermaid
+--8<-- "docs/core/event_handler/bedrock_agents.mermaid"
+```
+</center>
+
+## Key features
+
+* Minimal boilerplate to build Agents for Amazon Bedrock
+* Automatic generation of [OpenAPI schemas](https://www.openapis.org/){target="_blank"} from your business logic code
+* Built-in data validation for requests and responses
+* Similar experience when authoring [REST](api_gateway.md){target="_blank"} and [GraphQL APIs](appsync.md){target="_blank"}
```

Comment:
I'm not sure if I agree with that. We don't have DataValidation, Middleware and other things when build GraphQL APIs, the only similar thing is routing. Maybe REST and HTTP APIs?

Reply:
Changed to "REST and HTTP APIs"